package.json

```diff
@@ -61,10 +61,10 @@
   "dependencies": {
     "@oclif/command": "^1",
     "@oclif/plugin-help": "^2",
+    "@stoplight/json": "1.8.0",
     "@stoplight/json-ref-resolver": "^1.2",
-    "@stoplight/json": "1.x.x",
     "@stoplight/types": "4.x.x",
-    "@stoplight/yaml": "1.2.x",
+    "@stoplight/yaml": "2.1.0",
```

Comment:
Can we start using `^2.1.0`so that we allow 2.1.1 and 2.1.2 without needing to changew the package file?